services/src/apidoc.rs

```diff
@@ -131,6 +134,12 @@ use utoipa::{Modify, OpenApi};
             LayerCollectionListing,
             Property,
             CollectionItem,
+            AddLayer,
```

Comment:
you also need to add this stuff to the pro/apidocs

Reply:
[5b2a2e7](https://github.com/geo-engine/geoengine/pull/676/commits/5b2a2e71c4190aa6a8fc37ee559e4b7fadbace21)